### PR TITLE
Remove unused `0007-README.patch`.

### PR DESCRIPTION
**Pull request by mandrigin**

### CODE REVIEW

**Review by dshulyak — APPROVED** [review]

File: _assets/patches/geth/README.md [hunk 1]
```diff
@@ -16,13 +16,24 @@ Instructions for creating a patch from the command line:
...
+1. Remove tweak the patch file.
+2. Run `make dep-ensure` to re-apply patches.
+
+# Removing patches
+
+1. Remove the patch file
+2. Remove the link from [this README] (./README.md)
+3. Run `make dep-ensure` to re-apply patches.
+
+
```
Comment (dshulyak) [inline _assets/patches/geth/README.md]: why 2 lines?)

Comment (mandrigin) [reply _assets/patches/geth/README.md]: ¯\\_(ツ)_/¯
fixed tho

File: _assets/patches/geth/README.md [hunk 2]
```diff
@@ -16,13 +16,23 @@ Instructions for creating a patch from the command line:
     1. If you already have a commit that represents the change, find its SHA1 (e.g. `$COMMIT_SHA1`) and do `git diff $COMMIT_SHA1 > file.patch`
     1. If the files are staged, do `git diff --cached > file.patch`
 
+# Updating patches
+
+1. Remove tweak the patch file.
+2. Run `make dep-ensure` to re-apply patches.
```
Comment (themue) [inline _assets/patches/geth/README.md]: Nitpicking: Like above let markdown do the numbering. 😬

**Review by themue — APPROVED** [review]
Only one small wish, not really needed.

File: _assets/patches/geth/README.md (hunk 2 above)
Comment (mandrigin) [reply _assets/patches/geth/README.md]: ah, I tend to look at md files as sources, not after rendering...

Comment (mandrigin) [reply _assets/patches/geth/README.md]: fixed for consistency reasons

**Pull request merged**